code/Test_Definitions/NumberVerification_verify.feature

```diff
@@ -0,0 +1,132 @@
+    When the HTTPS "POST" request is sent
+    And the mobile connection is associated with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1
+    And the request body has NO the field phoneNumber
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And the response header "Content-Type" is "application/json"
+    And the response body complies with the OAS schema at "/components/schemas/SendCodeResponse"
+    Then the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+
+  @NumberVerification_verify201_missing_scope
+  Scenario:  verify phone number with valid access token but scope number-verification:verify is missing
+    Given they use the base url over a mobile connection
+    And the resource is "/verify"
+    And they acquired a valid access token associated with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1 through OIDC authorization code flow
+    And none of the scopes associated with the access token is number-verification:verify
+    When the HTTPS "POST" request is sent
+    And the mobile connection is associated with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1
+    And the request body has NO the field phoneNumber
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And the response header "Content-Type" is "application/json"
+    And the response body complies with the OAS schema at "/components/schemas/SendCodeResponse"
+    Then the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" is "Request not authenticated due to missing, invalid, or expired credentials."
+
+  @NumberVerification_verify202_expired_access_token
+  Scenario:  verify phone number with expired access token
+    Given they use the base url over a mobile connection
+    And the resource is "/verify"
+    And they acquired a valid access token associated with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1 through OIDC authorization code flow or CIBA
+    And one of the scopes associated with the access token is number-verification:verify
+    When the HTTPS "POST" request is sent
+    And the access token has expired
+    And the mobile connection is associated with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1
+    And the request body has NO the field phoneNumber
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And the response header "Content-Type" is "application/json"
+    And the response body complies with the OAS schema at "/components/schemas/SendCodeResponse"
+    Then the response property "$.status" is 401
+    And the response property "$.code" is "AUTHENTICATION_REQUIRED"
+    And the response property "$.message" is "New authentication is required."
```

Comment:
This message `New authentication is required` isn't in the spec.

Reply:
I think I took that from https://github.com/camaraproject/Commonalities/blob/main/artifacts/CAMARA_common.yaml

Reply:
Ohh, should it be reflected in the spec?

Reply:
arghhh probably yes !
As we targeting a stable release we need to provide this one. 

@AxelNennker you think I add it in #123 ?

Reply:
@AxelNennker @trehman-gsma Added in the specification.